app/src/main/java/com/example/project/model/NearbySearchResult.kt

```diff
@@ -1,6 +1,6 @@
 package com.example.project.model
 
-data class NearbySearch(
+data class NearbySearchResult(
     val html_attributions: List<Any>,
     val next_page_token: String,
     val results: List<Result>,
```

Comment:
This is what actually should be changed (from `Result` to `NearbySearchResult`). The reason is that Result is a pretty general name and it's hard for developers and reviewers who are not familiar with this project to fully understand what it stands for. Try to be semantic.